conftest.py

```diff
@@ -0,0 +1,235 @@
+import csv
+import json
+import os
+from pathlib import Path
+from typing import Iterator
+from unittest.mock import patch
+
+import boto3 as boto3
+import pytest as pytest
+import requests as requests
+from botocore.config import Config
+from requests.adapters import HTTPAdapter, Retry
+from tests import DEFAULT_ACCOUNT_ID
+
+from moto import settings
+from moto.core.models import BaseMockAWS
+
+BASE_PATH = os.path.join(os.path.dirname(__file__), "../../target/reports")
+
+FNAME_RAW_DATA_CSV = os.path.join(BASE_PATH, "metric_data_raw.csv")
+
+# overriding ENVs + settings to use LocalStack as endpoint
+os.environ["TEST_SERVER_MODE_ENDPOINT"] = "http://localhost:4566"
+os.environ["MOTO_CALL_RESET_API"] = "false"
+settings.TEST_SERVER_MODE = True
+
+
+@pytest.fixture(scope="session", autouse=True)
+def default_localstack_client_config_fixture() -> Iterator[None]:
+    def localstack_client(*args, org=boto3.client, **kwargs):
+        # kwargs["endpoint_url"] = 'http://localhost:4566' # TODO could also use fixture to override endpoint_url
+        config: Config = kwargs.get("config")
+        max_retry_config = Config(retries={"max_attempts": 1, "total_max_attempts": 2})
```

Comment:
Did you run into cases where the retry made it more stable or did you add this just-in-case? :thinking:

Reply:
Yes, actually. The first test runs took way longer, due to a lot of retries for requests that were failing on LocalStack.